core/state/statedb.go

```diff
@@ -102,7 +102,7 @@ type StateDB struct {
 	// Journal of state modifications. This is the backbone of
 	// Snapshot and RevertToSnapshot.
 	journal        *journal
-	validRevisions []revision
+	ValidRevisions []revision
```

Comment:
why was this made global? dont see it being used anywhere

Reply:
I made it public while debugging something, and forgot to make it private again. Good catch. I'll change it back.

Reply:
fixed